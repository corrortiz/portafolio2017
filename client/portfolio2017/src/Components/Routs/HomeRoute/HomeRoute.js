import React from 'react';

const HomeRoute = () => {
  return (
    <div>
      <h1>Home Rout, Papa</h1>
    </div>
  );
};

export default HomeRoute;
