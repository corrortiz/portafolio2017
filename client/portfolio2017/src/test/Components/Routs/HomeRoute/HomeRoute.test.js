// @flow
import React from 'react';
import { shallow } from 'enzyme';
import HomeRoute from '../../../../Components/Routs/HomeRoute/HomeRoute';

it('should compare a snapshot of HomeRoute', () => {
  const wrapper = shallow(<HomeRoute />);
  expect(wrapper).toMatchSnapshot();
});
