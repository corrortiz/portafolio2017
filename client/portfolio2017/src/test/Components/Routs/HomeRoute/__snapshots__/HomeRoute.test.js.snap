// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compare a snapshot of HomeRoute 1`] = `
<div>
  <Connect(NameWrappedComponent) />
  <Connect(NameWrappedComponent) />
  <Connect(NameWrappedComponent) />
  <Connect(NameWrappedComponent) />
</div>
`;
